app/models/user.rb

```diff
@@ -5,8 +5,15 @@ class User < ActiveRecord::Base
   has_many :storages, inverse_of: :user
 
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :omniauthable and
+  # :confirmable, :lockable, :omniauthable and
   # :registerable
-  devise :database_authenticatable,
+  devise :database_authenticatable, :timeoutable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  def self.from_saml_callback(response_attrs)
+    email = response_attrs[:va_eauth_emailaddress]
+    Rails.logger.info("email: #{email}")
```

Comment:
Is there more context that can be provided on how this info call should be used?

Reply:
Nope. It's a leftover from debugging.
Will remove